### ISSUE TITLE
New volto projects are broken

### ISSUE THREAD

**Issue by avoinea** [plone/create-volto-app#33]
How to reproduce:

```
create-volto-app my-volto-app
cd my-volto-app
yarn build
```

Output:
```
yarn run v1.22.5
$ razzle build
Creating an optimized production build...
Compiling client...
Failed to compile.
./src/client.js
Error: Cannot find module '@babel/plugin-proposal-function-bind' from '/joint/volto/my-volto-app'
    at Array.map (<anonymous>)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Possible solution:
- Update [yarn.lock](https://github.com/plone/create-volto-app/blob/master/volto-starter-kit/yarn.lock)

Refs https://github.com/plone/volto/pull/1896

**Comment by nileshgulia1** [comment]
Related #18 #19

**Comment by avoinea** [comment]
Fixed by create-volto-app#6.3.0
```
$ npm i -g @plone/create-volto-app
$ create-volto-app --version
6.3.0
$ create-volto-app my-volto-app
$ cd my-volto-app
$ yarn build
```
